### PR TITLE
if source folder does not exist, still allow to move to new folder

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
This is useful if you have switched to use the external sdcard and then removed it.

filepj61fw

Ref: #814

### CODE REVIEW

File: src/main/java/com/owncloud/android/ui/activity/StorageMigration.java [hunk 1]
```diff
@@ -205,15 +205,44 @@ protected void onPostExecute(Integer code) {
...
             if (mListener != null) {
                 mListener.onStorageMigrationFinished(succeed ? mStorageTarget : mStorageSource, succeed);
             }
         }
 
+        private void askToStillMove() {
+
+            new AlertDialog.Builder(mContext)
+                    .setMessage("Source directory not readable. Do you still want to change the storage path to "
+                            + mStorageTarget + "? Note: all data will have to be downloaded again.")
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/activity/StorageMigration.java]: Message needs to be extracted to string.xml :)

Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/activity/StorageMigration.java]: Source directory not readable - could be used as a title :)

Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/ui/activity/StorageMigration.java]: filejvnj84

**Comment by mario** [comment]
👍

**Comment by AndyScherzinger** [comment]
👍

**Pull request merged**